docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -887,6 +962,21 @@ public void producerError(Message<?> message) {
 
 [#scs-sb-headers]
 ===== Service Bus message headers
+The following table illustrates how Spring message headers are mapped to Service Bus message headers and properties. When create a message, developers can specify the header or property of a Service Bus message by below constants.
```

Comment:
Could you check if we have had similar content already? if so we can remove it here.